assets/dev/js/editor/elements/views/column.js

```diff
@@ -229,15 +230,11 @@ ColumnView = BaseElementView.extend( {
 			currentElementClass: 'elementor-html5dnd-current-element',
 			placeholderClass: 'elementor-sortable-placeholder elementor-widget-placeholder',
 			hasDraggingOnChildClass: 'elementor-dragging-on-child',
-			getDropContainer: () => this.getContainer(),
-			getDropIndex,
 			onDropping: ( side, event ) => {
-				event.stopPropagation();
-
-				// Triggering drag end manually, since it won't fired above iframe
-				elementor.getPreviewView().onPanelElementDragEnd();
-
-				this.addElementFromPanel( { at: getDropIndex( side, event ) } );
+				this.getContainer().view.onDropEvent(
```

Comment:
Do you need that? You're already inside the view

```suggestion
				this.onDropEvent(
```

Reply:
You're right. Fixed.